spec/support/pages/teacher_admin_page.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../pages/page'
+module Pages
+  class TeacherAdminPage < Page
+
+    def visit_page
+      visit '/admin/teachers'
+      self
+    end
+
+    def create_teacher(grove_name)
+      click_on "New Teacher"
+      fill_in "Name", with: "Bart Simpson"
+      fill_in "Email", with: "teacher@example.com"
+      click_on "Save"
+      page.has_content? "Teacher Saved"
+    end
```

Comment:
It just indicates the form submitted correctly and hides the implementation details of that in the method instead of needing to bother the caller with knowing what the successful form submission looks like. 

> On Jul 13, 2016, at 9:06 AM, Erinna Chen <notifications@github.com> wrote:
> 
> In spec/support/pages/teacher_admin_page.rb:
> 
> > +require_relative '../pages/page'
> > +module Pages
> > +  class TeacherAdminPage < Page
> > +
> > +    def visit_page
> > +      visit '/admin/teachers'
> > +      self
> > +    end
> > +
> > +    def create_teacher(grove_name)
> > +      click_on "New Teacher"
> > +      fill_in "Name", with: "Bart Simpson"
> > +      fill_in "Email", with: "teacher@example.com"
> > +      click_on "Save"
> > +      page.has_content? "Teacher Saved"
> > +    end
> Is there a reason to check content in the page versus in the test?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>